packages/Snowflake/Snowflake.0.02.02/opam

```diff
@@ -6,8 +6,8 @@ license: "LGPL-3.0-only with OCaml-LGPL-linking-exception"
 homepage: "https://gitlab.com/boreal-ldd/snowflake"
 bug-reports: "https://gitlab.com/boreal-ldd/snowflake"
 depends: [
-	"ocaml" {>= "4.08"}
-	"GuaCaml" {>= "0.03"}
+	"ocaml" {>= "4.08" & < "5.0"}
```

Comment:
This package seems to compile just fine on 5.0. Only some previous versions of GuaCaml failed
```suggestion
	"ocaml" {>= "4.08"}
```